neurom/io/utils.py

```diff
@@ -115,6 +116,16 @@ def get_files_by_path(path):
 
 def load_neuron(handle, reader=None):
     """Build section trees from an h5 or swc file."""
+    if isinstance(handle, (morphio.Morphology, morphio.mut.Morphology)):
+        fd, temp_file = tempfile.mkstemp(str(uuid.uuid4()) + '.h5', prefix='neurom-')
```

Comment:
Rather a question. Should we delete this `temp_file` after it's been used? According to [tempfile.mkstemp](https://docs.python.org/3/library/tempfile.html#tempfile.mkstemp) the user is responsible for its removal. We also produce temp files in `utils._get_file`. Should we clean them there also?

Reply:
Seconded, temp files should be cleaned up